scene/3d/label_3d.cpp

```diff
@@ -791,12 +791,17 @@ Ref<Font> Label3D::_get_font_or_default() const {
 		return font_override;
 	}
 
-	StringName theme_name = "font";
+	const StringName theme_name = "font";
 	List<StringName> theme_types;
 	ThemeDB::get_singleton()->get_native_type_dependencies(get_class_name(), &theme_types);
 
 	ThemeContext *global_context = ThemeDB::get_singleton()->get_default_theme_context();
-	for (const Ref<Theme> &theme : global_context->get_themes()) {
+	List<Ref<Theme>> themes = global_context->get_themes();
+	if (Engine::get_singleton()->is_editor_hint()) {
+		themes.push_front(ThemeDB::get_singleton()->get_project_theme()); // Preview in the editor properly.
```

Comment:
>// Preview in the editor properly.

Not sure if this comment is necessary

Reply:
To _me_ it feels weird that it has to be done this way. I can remove the comment though.

Reply:
I'm content either way